### PR TITLE
555 contacts action

### PR DESCRIPTION
**Pull request by Bhupesh-mfsi**
Task : #555 
Figma : https://www.figma.com/file/SqAsFElnxNzP69pic9RQah/dm3-embedded?type=design&node-id=85-2636&mode=design&t=lXGNgL8G7B3a8MDI-0

### CODE REVIEW

File: packages/messenger-widget/src/components/Contacts/bl.ts [hunk 1]
```diff
@@ -18,11 +18,14 @@ export function MouseOver(
     className: string,
```
Comment (AlexNi245) [inline packages/messenger-widget/src/components/Contacts/bl.ts:18]: Please give the file a proper name

File: packages/messenger-widget/src/components/Contacts/bl.ts [hunk 2]
```diff
@@ -86,7 +89,7 @@ export const onContactSelected = (
     // add click css
     event.currentTarget.classList.add(classTwo);
     const actionItem: any = document.getElementById('contact-' + index);
-    actionItem.style.display = 'none';
+    // actionItem.style.display = 'none';
```
Comment (AlexNi245) [inline packages/messenger-widget/src/components/Contacts/bl.ts:92]: Do you intend to keep using that var ? Otherwiese please remove the comment

File: packages/messenger-widget/src/components/Contacts/bl.ts [hunk 3]
```diff
@@ -192,9 +195,13 @@ export const setContactSelectedFromCache = (
     state: GlobalState,
     cacheContacts: ContactPreview[],
 ): number | null => {
-    const name = state.accounts.selectedContact?.account.ensName;
+    const key =
+        state.accounts.selectedContact?.account.profile?.publicEncryptionKey;
     for (let index = 0; index < cacheContacts.length; index++) {
```
Comment (AlexNi245) [inline packages/messenger-widget/src/components/Contacts/bl.ts:200]: If there is no good reason to use for() please use a higher order function. I think .filter() would work for that case

**Review by AlexNi245 — CHANGES_REQUESTED** [review]
see comments

File: packages/messenger-widget/src/components/Contacts/bl.ts (hunk 2 above)
Comment (Bhupesh-mfsi) [reply packages/messenger-widget/src/components/Contacts/bl.ts:92]: Oh yes, I will remove that

File: packages/messenger-widget/src/components/Contacts/bl.ts (hunk 3 above)
Comment (Bhupesh-mfsi) [reply packages/messenger-widget/src/components/Contacts/bl.ts]: Ok, will replace for loop with findIndex high order function as I needed index here.

File: packages/messenger-widget/src/components/Contacts/bl.ts (hunk 1 above)
Comment (AlexNi245) [reply packages/messenger-widget/src/components/Contacts/bl.ts:18]: okay if you've agreed on that with @hai-ko it's fine for me

**Review by AlexNi245 — APPROVED** [review]

**Pull request merged**